apply.js

```diff
@@ -42,9 +42,16 @@ const patches = [
     description: "todo",
     files: {
       js: ["redirectToBoard/script.js"],
-      css: ["redirectToBoard/styles.css"]
+      css: ["redirectToBoard/styles.css"],
     },
-  }
+  },
+  {
+    name: "Hide Help On Dashboard",
+    description: "Hide irritating blue help block on dashboard",
```

Comment:
It's blue only in eduVULCAN, Dziennik Vulcan has it red. Also, I'd descriptions to not be as opinionated.